presto-hive/src/main/java/com/facebook/presto/hive/HiveTableProperties.java

```diff
@@ -55,7 +58,14 @@ public HiveTableProperties(TypeManager typeManager, HiveClientConfig config)
                         false,
                         value -> ImmutableList.copyOf(((List<String>) value).stream()
                                 .map(name -> name.toLowerCase(ENGLISH))
-                                .collect(Collectors.toList()))));
+                                .collect(Collectors.toList()))),
+                new PropertyMetadata<>(
+                        SERDE_PARAMETERS,
+                        "SerDe properties",
+                        typeManager.getParameterizedType(MAP, ImmutableList.of(VARCHAR.getTypeSignature(), VARCHAR.getTypeSignature()), ImmutableList.of()),
```

Comment:
shouldn't it be like each separate serde parameter is one separate table property? Can you get the list of the known parameter names from somewhere?